frontend/src/components/Map/styles.module.css

```diff
@@ -13,13 +13,18 @@
 }
 
 .mask {
-  background: rgba(0, 0, 0, 0.40);
-  z-index: 2
+  z-index: 2;
+}
+
+.opaque {
+  filter: saturate(0) contrast(0.4) brightness(0.4);
```

Comment:
Just letting you know since cross-browser compatibility came up recently, [filters](https://caniuse.com/#feat=css-filters) have a history of not working in many browsers - they never were supported in IE, and only got full support in Edge recently.

There's no foolproof way to solve this that I can think of, but let's keep it in mind